docs/book/src/clusteropenstack/configuration.md

```diff
@@ -109,6 +109,8 @@ Note: you need to set `clusterctl.cluster.x-k8s.io/move` label for the secret cr
 
 The availability zone names must be exposed as an environment variable `OPENSTACK_FAILURE_DOMAIN`.
 
+By default, if `Availability zone` is not given, all `Availability zone` that defined in openstack will be a candidate to provision from, If administrator credential is used then `internal` Availability zone which is internal only Availability zone inside `nova` will be returned and can cause potential problem, see [PR 1165](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1165) for further information. So we highly recommend to set `Availability zone` explicitly.
```

Comment:
Nice. Do we document anywhere that admin credentials aren't recommended?

Reply:
I don't think we do, I can find a place to update